chukwuemekachm/M-Tracker-React-Client
# #160296432 Completes user delete request feature

#### What does this PR do?
- This PR completes user delete request feature
#### Description of Task to be completed?
- Add deleteRequestAsync to userRequestActions
- Refactor reducers
- Write tests for refactored reducers
- Implement delete request on ViewSingleRequestPage.jsx
#### How should this be manually tested?
- Clone the repo
- Startup command line/Terminal
- Navigate to the project's directory
- Run `npm install`
- Run `npm start` or `npm run dev`
- Your default browser will be opened if you used `npm run dev`
- Login or signup(navigate to `/login` or `/signup`)
- Click on any request on your dashboard or create a new request and click it
- The single request page will appear
- Click the red delete icon on the top of the page
- Observe the message from the toastr
#### Production URL
- [heroku](https://m-tracker-react.herokuapp.com)
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#160296432](https://www.pivotaltracker.com/story/show/160296432)
#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-10-02 at 2 34 49 am" src="https://user-images.githubusercontent.com/33798252/46324637-6909be00-c5ec-11e8-9dd8-3f0ecc46d723.png">

#### Questions:
- None
